Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj

```diff
@@ -13,9 +13,9 @@
   <ItemGroup>
     <PackageReference Include="Plugin.BLE" Version="2.1.3" />
     <PackageReference Include="sqlite-net-pcl" Version="1.8.116" />
-    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2012" />  
-    <PackageReference Include="Xamarin.Essentials" Version="1.6.1" />  
-    <PackageReference Include="Xamarin.Forms.Maps" Version="5.0.0.2012" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2478" />  
+    <PackageReference Include="Xamarin.Essentials" Version="1.7.3" />  
+    <PackageReference Include="Xamarin.Forms.Maps" Version="5.0.0.2478" />
```

Comment:
If you want to change the Xamarin-Versions, you not only need to adapt the iOS-Setup, but also the android-Setup accordingly.

Reply:
done